sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultStep.java

```diff
@@ -143,16 +140,6 @@ public Status getStatus() {
         }
     }
 
-    @Override
-    public List<Element<?>> getChildren() {
-        return Collections.emptyList();
-    }
-
-    @Override
```

Comment:
I am not 100% sure whether it is a good idea to get rid of interrupts from steps. I think every element should be interruptible. Interrupt is similar to status. getStatus()/update() ~ pause()/proceed().